api/cotrip_app/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('city/', views.CityList.as_view(), name='city_list'),
+    path('city/<int:pk>', views.CityDetail.as_view(), name='city_detail'),
+    path('city/new', views.CityList.as_view(), name='city_create'),
+    path('city/<int:pk>/edit', views.CityDetail.as_view(), name='city_edit'),
+    path('city/<int:pk>/delete',
```

Comment:
I don't think we want people to be able to create, update or delete these through the API